### PR TITLE
Cannot make toolbar appear on empty document (immediately after gaining focus)

### LINKED ISSUE #29: Toolbar doesn't appear after a long press while gaining focus
Steps to reproduce

1. Empty document in example (select all - delete)
2. Open example menu, then close it
3. long press on document -> nothing appears (toolbar should appear)

Workaround is to type some text and then the long press (or the double tap) will cause the toolbar to appear.

This is particularly annoying for users who wish to paste something in a new document.
Their impression is that they cannot paste things. They end up not using the product

Video

### PR DESCRIPTION
**Pull request by amantoux**
Fixes #29 

-------

Before fix (go to menu once document is open, emulates opening an empty document and setting focus)

https://user-images.githubusercontent.com/17318211/186650849-74b992b9-dea5-419a-bbe9-e3778ada3ddc.mov

After fix

https://user-images.githubusercontent.com/17318211/186650873-f787b8a9-3a67-4c4d-b504-e238fc7f669d.mov

### CODE REVIEW

**Review by Amir-P — APPROVED** [review]
LGTM!

**Comment by Amir-P** [comment]
Just a question: Why did you re-add removed TODO? @amantoux

**Comment by amantoux** [comment]
> Just a question: Why did you re-add removed TODOs? @amantoux

Did I? last commit removes them I believe

**Comment by amantoux** [comment]
I've added the same behavior with double tap

_NB: We need to review the whole Editor renderer - widget in order to stick to Flutter latest Editable - EditableText arch; particularly with regards to focus management and user interactions_

File: packages/fleather/lib/src/widgets/editor.dart [hunk 1]
```diff
@@ -26,6 +26,9 @@ import 'text_line.dart';
 import 'text_selection.dart';
 import 'theme.dart';
 
+// TODO: add test - paste on empty document (should not crash on assert)
```
Comment (Amir-P) [inline packages/fleather/lib/src/widgets/editor.dart]: Here you've added them again. @amantoux

File: packages/fleather/lib/src/rendering/editor.dart [hunk 2]
```diff
@@ -608,7 +608,8 @@ class RenderEditor extends RenderEditableContainerBox
     // as well.
     if (selectionChanged ||
         (cause == SelectionChangedCause.longPress ||
-            cause == SelectionChangedCause.keyboard)) {
```
Comment (Amir-P) [inline packages/fleather/lib/src/rendering/editor.dart]: I think it's better to extract this into an extension method on `SelectionChangedCause.longPress`. WDYT? @amantoux

Comment (amantoux) [reply packages/fleather/lib/src/rendering/editor.dart]: @Amir-P what do you mean?

Comment (Amir-P) [reply packages/fleather/lib/src/rendering/editor.dart]: Something like this:
```
Extension on SelectionChangedCause {
  bool get shouldSendEvent => this == SelectionChangedCause.longPress || ...
}
```
WDYT? @amantoux

Comment (amantoux) [reply packages/fleather/lib/src/rendering/editor.dart]: Ok, I'm not sure it will be needed though it the future
